[[987136, "Traceback (most recent call last):\r\n  File \"B:\\Sasha\\01_Sync\\Exercises\\Yandex\\\u2568\u042f\u2564\u0410\u2568\u255b\u2568\u2561\u2568\u2551\u2564\u0412\u2564\u041b\\Test system\\program_testing\\source_solution\\solution.py\", line 1, in <module>\r\n    gj\r\nNameError: name 'gj' is not defined", null, 0.002003]]